recipes/gprofiler/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "gprofiler" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}-pkg
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/g/gprofiler-official/gprofiler-official-{{ version }}.tar.gz
+  sha256: 5015b47f10fbdcb59c57e342e815c9c07afbe57cd3984154f75b845ddef2445d
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ name }}
+    script: build_base.sh
+    build:
+      noarch: python
+    requirements:
+      host:
+        - pip
+        - python {{ python_min }}
+        - setuptools
+      run:
+        - python >={{ python_min }}
+        - requests
+    test:
+      imports:
+        - gprofiler
+      commands:
+        - pip check
+      requires:
+        - pip
+        - python {{ python_min }}
+
+  # Name used for distribution on PyPI
+  - name: {{ name }}-official
+    build:
+      noarch: generic
+    requirements:
+      run:
+        - {{ pin_subpackage(name, max_pin="x.x.x") }}
```

Comment:
Can you explain the reason for this pinning?  Most of the this is needed to preserver ABI compatibility for shared native libraries, which this package is not.